src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -438,6 +438,47 @@ def test_group_deployment(self, resource_group):
             self.check('[0].resourceGroup', '{rg}')
         ])
 
+    @ResourceGroupPreparer(name_prefix='cli_test_deployment')
+    def test_group_deployment(self, resource_group):
```

Comment:
Both two tests are live_only. Right?

Reply:
Thanks for reminder! It is duplicated and should be deleted!